src/services/PopularCitiesServise.js

```diff
@@ -0,0 +1,35 @@
+import instance from '../utilities/axios/axios';
+
+const DEFAULT_PAGE = 1;
+const DEFAULT_PAGE_SIZE = 10;
+const DEFAULT_SORT_ORDER = 'POPULARITY_DESC';
+const DEFAULT_ENDPOINT = '/cities';
+
+async function fetchPopularCities({
+  endpoint = DEFAULT_ENDPOINT,
+  page = DEFAULT_PAGE,
+  pageSize = DEFAULT_PAGE_SIZE,
+  sortOrder = DEFAULT_SORT_ORDER,
+  onlyWithGuides = false,
+} = {}) {
+  const params = {
+    page,
+    pageSize,
+    sortOrder,
+    onlyWithGuides,
+  };
+
+  try {
+    const response = await instance.get(endpoint, { params });
+    if (response.data && response.data.data.items) {
```

Comment:
Duplicated checks might be replaced with options chaining operator

```suggestion
    if (response?.data?.data?.items) {
```

Reply:
NEXT LVL 🔥🔥🔥